rskj-core/src/test/java/co/rsk/mine/MinerUtilsTest.java

```diff
@@ -288,4 +293,30 @@ public void getAllTransactionsCheckOrderTest() {
         Assert.assertEquals(res.get(7).getGasPrice(), Coin.valueOf(100));
         Assert.assertEquals(res.get(8).getGasPrice(), Coin.valueOf(1));
     }
+
+    @Test
+    public void filterTransactions_getNonceThrowsNodeReferenceException_throwsRuntimeException() {
+        // Given
+        RskAddress senderAddressMock = new RskAddress(new byte[20]);
+        Transaction txMock = Tx.create(config, 0, 50000, 5, 0, 0, 0);
```

Comment:
maybe we could extract this to a method? like `getTestTransaction` or `getMockTransaction` since I've seen it used one other time in the same test class (and since we don't care about the values, we could just call it without parameters).
I'd also consider naming the parameters we pass (value, gasLimit, nonce, etc.) but maybe it's not necessary